app/src/main/scala/org/alephium/explorer/persistence/queries/BlockQueries.scala

```diff
@@ -69,15 +69,14 @@ object BlockQueries extends StrictLogging {
       blocks  <- DBIOAction.sequence(headers.map(buildBlockEntryAction))
     } yield blocks.headOption
 
-  def getBlockHeaderAction(hash: BlockEntry.Hash): DBActionR[Option[BlockHeader]] = {
+  def getBlockHeaderAction(hash: BlockEntry.Hash): DBActionR[Option[BlockHeader]] =
     sql"""
        |SELECT *
        |FROM #$block_headers
-       |WHERE hash = '\x#${hash.toString()}'
+       |WHERE hash = decode(${hash.toString()}, 'hex')
```

Comment:
actually you can directly have `WHERE hash = $hash` as we have [the corresponding SetParameter](https://github.com/alephium/explorer-backend/blob/28e2930c497cc1219fe68f5ef05d3387f21b9885/app/src/main/scala/org/alephium/explorer/persistence/schema/CustomSetParameter.scala#L51-L54)